### ISSUE TITLE
GRUB out of memory error when using nvidia image

### ISSUE THREAD

**Issue by kgadberry** [ublue-os/hwe#272]
I'm unable to use my graphics card in any ublue-nvidia image.

GRUB loads and works fine, but every time I try to boot with a display (any display) connected to one of the GPU's outputs, I get `error: ../../grub-core/kern/mm.c:552: out of memory`. 

I've tried Bluefin, Bazzite, and Aurora, :39, :40 :gts, :latest, all of them have the same behavior.

I can use any non-nvidia image with the display connected to the GPU, and I can only use an nvidia image with the display connected to the IGPU.

Completely reinstalling from a clean ISO changes nothing. Behavior is consistent regardless of what image I'm rebasing from. Running `rpm-ostree reset` obviously does nothing.
I've tried the oldest image I can rebase to without a PKE error which is `40-20240703` with the same results.

None of the options available to me in the UEFI settings affects it.

Output of `rpm-ostree status`:
```
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: no runs since boot
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx-nvidia:40
                   Digest: sha256:416751631c624f15659e3119158f7487d26bbe3c8884c51a3511799b8227bce1
                  Version: 40.20240802.0 (2024-08-02T04:53:35Z)
          LayeredPackages: proton-vpn-gnome-desktop
            LocalPackages: protonvpn-stable-release-1.0.1-2.noarch

  ostree-unverified-registry:ghcr.io/ublue-os/bazzite-nvidia:stable
                   Digest: sha256:34fe01d55e5707c77f4aea3f806d27baa5ff788626bfbb57af2753521274401f
                  Version: 40.20240803.0 (2024-08-04T06:37:14Z)
          LayeredPackages: proton-vpn-gnome-desktop
            LocalPackages: protonvpn-stable-release-1.0.1-2.noarch
```

Output of `neofetch`:

![image](https://github.com/user-attachments/assets/7df2e77c-6b1e-4972-a448-9199cc4bd475)

**Comment by m2Giles** [comment]
See if anything in here helps

https://discussion.fedoraproject.org/t/grub-crashing-in-mm-c-when-trying-to-boot-fedora-39-live-usb/104614/15

**Comment by kgadberry** [comment]
Because I'm not using a laptop there aren't really any options involving graphics memory, I've tried setting everything that I can find differently and nothing helps. Removing all but one RAM module also does nothing.

**Comment by m2Giles** [comment]
You have an iGPU so you might have some settings for that

**Comment by kgadberry** [comment]
I've spent a few hours combing through the settings. I've changed all of them that look even remotely relevant. Nothing helps.

**Comment by kgadberry** [comment]
I cleared the CMOS and suddenly my secondary display (which wasn't working either, previously) is now working from GPU output. I returned all UEFI settings back to what they were before, and it still works.
My primary display, however, just goes blank after GRUB launches the operating system... unless it's connected to the IGPU output.

In any case, I suppose I can best assume this is a hardware issue and close this.
